scenes/quests/lore_quests/quest_001/3_stealth_level/stealth_level.tscn

```diff
@@ -855,6 +863,10 @@ curve = SubResource("Curve2D_vnsq3")
 [node name="Guard2-GoingInCircles" parent="EnemyGuards" node_paths=PackedStringArray("patrol_path") instance=ExtResource("8_lbv6i")]
 position = Vector2(1567, 101.949)
 sprite_frames = ExtResource("23_5q0mi")
+alerted_sound_stream = ExtResource("24_agwy0")
+footsteps_sound_stream = ExtResource("25_dsau0")
+idle_sound_stream = ExtResource("26_w5e5c")
+alert_others_sound_stream = ExtResource("27_lp8w2")
```

Comment:
This works for now, but I wonder whether we might want to define an inherited scene, or a copy of the scene, to avoid having to remember to set these on each guard in the level. (Ditto InkDrinkers.) Specifically I imagine we may some day want to replace the asset on the guards; or replace the stomach-churning spitting noise on the InkDrinker.

Reply:
Yes, we are facing the same problems learners will have when using the template elements. I haven't tried inherited scenes myself yet. I have that pending after your feedback on projectiles.

It wasn't that bad to use copy/paste in the editor to do this PR. But is true that the same override applied everywhere is a lot of redundance and prone to errors. So yes, let's try inherited scenes and see how it goes.